src/sort-array.js

```diff
@@ -0,0 +1,23 @@
+export default sortArray
+
+/**
+ * Original Source: http://stackoverflow.com/questions/5476736/javascript-sort-array
```

Comment:
Could you link this to the specific answer that this is using? There's a `share` link below the answer.